File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio13.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio13 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o valor do salário: ");
        float salario = entrada.nextFloat();

        float aumento = salario * 15/100;
        float novoSalario = salario + aumento;
        System.out.println("Salário antigo: R$ " + salario+ "\nSalário reajustado (+15%): R$" +novoSalario);
    }
}

File: guanabara-c/lista01/exercicio13.c
#include <stdio.h>

int main(){
    float salario_atual, novo_salario, aumento;

    printf("Insira o salário do funcionário: ");
    scanf("%f", &salario_atual);

    aumento = salario_atual * 15 / 100;
    novo_salario = salario_atual + aumento;
    printf("Salário reajustado: R$ %.2f", novo_salario);

    return 1;
}
